chore: Bump gci to v0.11.0

codingllama opened the pull request:
Update to latest release.

* https://github.com/daixiang0/gci/releases/tag/v0.11.0

Note: updates to devbox to v0.10.1, which is the latest version on nixpkgs.

jakule reviewed (approved):
Are they any changes in formatting?

codingllama committed:
chore: Bump gci to v0.11.0

Update to latest release.

* https://github.com/daixiang0/gci/releases/tag/v0.11.0

codingllama commented:
FYI @mdwn

codingllama commented:
FYI reviewers: I split Dockerfile and devbox changes into separate commits (for easy backporting) and updated devbox.lock (it took a while to get my devbox running.)

> Are they any changes in formatting?

Nope, no changes.
